Assets/Scripts/Player/Player.cs

```diff
@@ -67,6 +67,12 @@ class Player : Entity
         /// <summary> True if the player has been hit by something damaging. </summary>
         protected bool hit;
 
+        public struct HealthBundle
```

Comment:
Move this into HealthManagerDisplay.cs, its main use is there and only needs to be referenced here for construction.